aten/src/ATen/core/dispatch/Dispatcher.h

```diff
@@ -149,6 +149,10 @@ class CAFFE2_API OperatorHandle final {
     return operatorIterator_->op.options();
   }
 
+  const DispatchTable& dispatchTable() const {
```

Comment:
Hm this exposes an implementation detail. Is there a way around this, e.g. friending the test or making it a unit test that doesn't use the Dispatcher but only the KeyExtractor?